src/exportImage.py

```diff
@@ -15,6 +15,16 @@ def get_zoom_resolution(zoom_to_resolution_dict, zoom_level):
         raise Exception(f'No such zoom level. got: {zoom_level}')
 
 
+def is_valid_zoom_for_area(resolution, tile_size, bbox):
+    top_right_lat = bbox[3]
+    top_right_lon = bbox[2]
+    bottom_left_lat = bbox[1]
+    bottom_left_lon = bbox[0]
+
+    # Check if bbox width and height are at least at the resolution of a pixle at the wanted zoom level
+    return (top_right_lon - bottom_left_lon) >= float(resolution) and (top_right_lat - bottom_left_lat) >= float(resolution)
```

Comment:
a. talk with me about this calculation
b. will it work on different parts of the world? (
check with lat and long of:
x1<0, x2<0
x1<0, x2>0
x1>0, x2>0
y1<0, y2<0
y1<0, y2>0
y1>0, y2>0